src/AppBundle/Controller/BlogController.php

```diff
@@ -44,7 +44,10 @@ public function indexAction($page)
         $posts = $paginator->paginate($query, $page, Post::NUM_ITEMS);
         $posts->setUsedRoute('blog_index_paginated');
 
-        return $this->render('blog/index.html.twig', array('posts' => $posts));
+        $response = $this->render('blog/index.html.twig', array('posts' => $posts));
+        $response->setSharedMaxAge(10); // cached for 10 seconds
```

Comment:
@sergiu-popa, using annotations for caching is a best practice:
```
Make your controller extend the FrameworkBundle base controller and use annotations to configure routing, caching and security whenever possible.
```

See http://symfony.com/doc/current/best_practices/controllers.html